Survey/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -15,8 +15,8 @@ struct LoginView: View {
     @State private var loginViewOpacity = 0.0
     @State private var isLoginFailed = false
     @State var editingMode: Bool = false
-    @State var email: String = ""
-    @State var password: String = ""
+
+    @StateObject var viewModel: LoginViewModel
```

Comment:
The `LoginView` doesn't create its own viewModel, instead the viewModel is passed when initializing the `LoginView`. So the proper property wrapper here is `@ObservedObject`, not `@StateObject`. For more, [this site](https://swiftuipropertywrappers.com) by __Donny Wals__ has detailed explanation the difference between them and other SwiftUI's property wrappers 🙏

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/43/commits/1aab549f658f6030d71ed30af0d2429788c73d4b)